releasenotes/notes/feat-vscf-and-hartree-as-blueprintcircuits-15167723fdd504d5.yaml

```diff
@@ -0,0 +1,35 @@
+---
+features:
+  - |
+    Implements both :class:`~qiskit_nature.second_q.circuit.library.initial_states.HartreeFock` and
+    :class:`~qiskit_nature.second_q.circuit.library.initial_states.VSCF` as subclasses of
+    :class:`~qiskit.circuit.library.BlueprintCircuit`. This allows the respective classes to be
```

Comment:
Actually, `BlueprintCircuit` is now resolvable as a link, but even on the published documentation of Terra one cannot navigate to this class' documentation. So this can be left as is.